include/boost/beast/core/detail/lean_tuple.hpp

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright (c) 2018 Damian Jarek (damian dot jarek93 at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+
+#ifndef BOOST_BEAST_DETAIL_LEAN_TUPLE_HPP
+#define BOOST_BEAST_DETAIL_LEAN_TUPLE_HPP
+
+#include <boost/mp11/integer_sequence.hpp>
+
+#if __cplusplus > 201703L
```

Comment:
This is wrong. You cannot assume this for *any* C++ language feature or C++ library feature. This is why we have Boost.Config. Because C++ implementations that have  `__cpluplus`  greater than 201703L (e.g. in C++2a standards mode) still do not have this feature implemented.